inc/saiport.h

```diff
@@ -745,7 +745,7 @@ typedef enum _sai_port_attr_t
      * @brief Hardware Lane list
      *
      * @type sai_u32_list_t
-     * @flags MANDATORY_ON_CREATE | CREATE_ONLY | KEY
```

Comment:
hw lane list identifies uniquely ports on a single switch, it must be a KEY, please revert

Reply:
is the error you getting from metadata in OA or syncd? can you past error?

Reply:
this fix will resolve that: https://github.com/Azure/sonic-sairedis/pull/865

Reply:
Sure. I'll the try your fix and hopefully close this PR thereafter. Thanks.